dependencies.rosinstall

```diff
@@ -32,4 +32,10 @@
 - git:
     local-name: message_serialization
     uri: 'https://github.com/swri-robotics/message_serialization.git'
-    version: c054e73dd8abc98b9aecef1ce96f4fec5394d890
\ No newline at end of file
+    version: c054e73dd8abc98b9aecef1ce96f4fec5394d890
+
+# Colorized mesh RViz plugin
+- git:
+    local-name: colorized_mesh_display
+    uri: 'https://github.com/marip8/colorized_mesh_display.git'
+    version: master

```

Comment:
Please use a git hash here instead of a named branch for better version control

Reply:
Alright Ill make that change.